README.md

```diff
@@ -33,23 +32,24 @@ Creates the `nasa/harmony-gdal-adapter-test` test image.
 ```bash
 bin/run-test
 ```
-The `run-test` script mounts `test-reports` and `coverage` directories and run
-the test script `tests/run_tests.sh` inside of a Docker test container.
+
+The `run-test` script mounts `test-reports` and `coverage` directories and runs
+the test script `tests/run_tests.sh` inside of the Docker test container.
 
 
 ## Test Locally
 
 ### Create isolated environment
 
 ```bash
-conda create --name hga python=3.8 --channel conda-forge
+conda create --name hga python=3.10 --channel conda-forge
```

Comment:
Good call. I also checked the worker container to see what the GDAL image is using (possibly what you also did). The container also has 3.10.